### PR TITLE
Adding examples and output generated by UPLC

### PR DESCRIPTION
**Pull request by SchmErik**
The uplc program contains several Plutus examples along with an
evaluator for Untypted Plutus. Although the examples are generated with
some notion of random generation, it's important for us to have a set of
static tests for repeatability.

Examples were generated using the uplc-example-gen.sh script.

### CODE REVIEW

**Comment by ehildenb** [comment]
This looks good, let's wait for #61 to merge, then merge this but with the updated `tests/failing` list, and a target `test-uplc-generated` which tries to run these tests (initially skipping them all because they're marked as failing).

**Comment by SchmErik** [comment]
> This looks good, let's wait for #61 to merge, then merge this but with the updated `tests/failing` list, and a target `test-uplc-generated` which tries to run these tests (initially skipping them all because they're marked as failing).

Ok, I'll make this adjustment after #61 gets merged

**Review by ehildenb — APPROVED** [review]

File: Jenkinsfile
Comment (ehildenb) [inline Jenkinsfile:29]:
```suggestion
            stage('Simple')        { steps { sh 'make test-simple -j4'        } }
            stage('uplc-examples') { steps { sh 'make test-uplc-examples -j4' } }
```

File: Makefile
Comment (ehildenb) [inline Makefile:194]: `test-uplc-examples` needs to be added to `.PHONY` (at the top of the file).

**Review by ehildenb — APPROVED** [review]

**Pull request merged**
